.github/workflows/main_backend.yml

```diff
@@ -0,0 +1,32 @@
+name: Build backend service ASP.NET CORE 3.1 APP
+
+on:
+  push:
+    branches: [ master ]
+    paths: 'GiEnJul/**'
```

Comment:
I think the backend needs to just exclude the **frontend folder.**

We might need to experiment a little with this.

Reply:
So either exclude the frontend folder it in the main_backend.yml
or refactor so the folder appears at the root of the project